eclipse-theia/vscode-builtin-extensions
# Uplift to use OVSX 0.8.1

Fixes: https://github.com/eclipse-theia/vscode-builtin-extensions/issues/114

The newer version of OVSX allows the use of a later version of `vsce` e.g. v2.15.0 or higher.

This allows us to comply to new checks and generate `vsix` packages with additional extension meta-data.

 #### How to test

1) Build (compiles and bundles) 
> yarn 
2) Create `vsix` files under `dist` folder
> yarn package-vsix:latest
3) Use a `theia` repository `plugins` folder to make the generated `vsix` load as plugins
e.g. 
> cd <theia-repo-wdir>
> rm -fr plugins/
> ln -s /home/$USER/git/vscode-builtin-extensions/dist plugins/
4) Open some e.g.,  `typescript` and `javascript` files to make sure the syntax highlighting and basic functionality offered by the extensions is present.
 
5) Remove the soft link created above
> rm plugins

6) Create a `plugins` folder and leave it empty.
7) Create an extension pack under the `dist` folder
> yarn create-extension-pack:latest
8) Start `theia` and from the `Extensions` view, install the extension pack i.e., `Install from VSIX...`
9) Once installed, Verify the extensions are listed under the `Extensions` view `INSTALLED` section 
10) Repeat the testing done in step 4, to validate the basic functionality offered by the extensions.
